packages/ckeditor5-html-support/src/datafilter.js

```diff
@@ -53,6 +53,9 @@ import '../theme/datafilter.css';
  *			}
  *		} );
  *
+ * To apply the information about allowed and disallowed attributes in custom integration plugin,
+ * use {@link module:html-support/datafilter~DataFilter#processViewAttributes} method.
```

Comment:
```suggestion
 * use the {@link module:html-support/datafilter~DataFilter#processViewAttributes `processViewAttributes()`} method.
```

Easier to read this way. Also, missed `the`.

(it may require reformatting due to max line length)